news/2019-03-31-beatmap-spotlights-winter-2019.md

```diff
@@ -0,0 +1,308 @@
+
+Overall this set is just fun and a nice addition to this season's Spotlights.
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/taiko/touhou.jpg)](https://osu.ppy.sh/beatmapsets/871329)
+
+From early winter, [komasy](https://osu.ppy.sh/users/1980256)'s high quality osu!taiko mapset of SDVX III's medley of all themes from "Touhou Youyoumu ~ Perfect Cherry Blossom" arrived to the ranked section, and quickly became a top favorite for a lot of players, and of course, for us too.
+
+With a wide range of 6 difficulties, from an intermediate Kantan to a very challenging top difficulty, komasy's 21st ranked mapset shows its creator excellence at mapping, not only because of the vast difficulties present in the mapset, but also because of the diversity of styles a mapper needs to know in order to achieve a map like this due to the nature of this medley.
+
+Touhou Youyoumu ULTIMATE MEDLEY is a track that features a lot of diverse Electronic music subgenres in a short amount of time, such as Psytrance or Speedcore, which komasy's mapping has translated neatly to the osu!taiko playfield through playability focused but interesting patterns that match the rhythmic complexity in the music perfectly, for both stamina and technical oriented players.
+
+Great music, great mapper... A mapset for everyone.
+If you are a fan of hype-overflowing maps, this is your map then, every section of the map is a new sensation!
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/taiko/feline.jpg)](https://osu.ppy.sh/beatmapsets/847253)
+
+From the creator of multiple breakcore charts as well as a multi-variety of most peculiar genres, [-Kazu-](https://osu.ppy.sh/users/920861) shows up in Winter Spotlights with his interpretation of Cranky's "Feline, The White...". Kazu went for a more unconventional approach this time around, though! He gave the creation a distinct purpose, a concept. Every bit of this beatmap can be correlated to a set of concepts, which is helped and represented by the song's pacing. Does it have upbeat jazz? Then give it high slider velocity! Does it have classic breakbeats? Then repeat the background pattern! So on and so forth.
+
+It does not end here, though. The beatmap itself is challenging on its own! Apart from the concept mapping, Kazu also made sure to squeeze every element of the music as much as possible, resulting in several mixed snaps (it's still breakcore) that will keep all players on their toes!
+
+All in all, for the sheer capacity of interpretation, the outstanding musical ear the mapper shows with this creation, and the beatmap itself being also incredibly fun to play, Feline, The White... earns its spot on these seasonal Spotlights very fairly!
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/taiko/give-it-up.jpg)](https://osu.ppy.sh/beatmapsets/618352)
```

Comment:
maybe add nofool and Lno ? not sure if they count as creators but they are in the tags of the mapset